daily-shoutout/app.py

```diff
@@ -55,11 +56,22 @@ def sendTweet(tweet_string):
     client.create_tweet(text = tweet_string)
 
 
+def returnResponse(response_string):
+    return{
+        "statusCode": 200,
+        "body": json.dumps(response_string)
+    }
+
+
 def lambda_handler(event, context):
```

Comment:
You could update the signature here to default to `None` and then make your call under `if __name__ == "__main__" cleaner:

```suggestion
def lambda_handler(event=None, context=None):
```

Reply:
I will have to test this idea.  I know that lambda EXPECTS an event and context from whatever is invoking it.  It seems more "right" to mock up the local invocations with dummy data than make the main function default to something..